ui/client/components/graph/node-modal/editors/expression/duration/TimeRangeEditor.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {useEffect, useState} from "react";
```

Comment:
semicolons

Reply:
because _data field is not recognized by ts, but it does exist!

Reply:
someone disabled githooks? ;)